metadata/labels.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metadata
+
+import "fmt"
+
+// Common constants
+const (
+	// appstudioLabelPrefix is the prefix of the application label
+	appstudioLabelPrefix = "appstudio.openshift.io"
```

Comment:
Can you rename the variable? This is not a prefix anymore. Change it to `rhtapDomain` or something similar. @johnbieren feel free to pitch in if you have a better proposal.

Reply:
I would change it to something like _applicationPrefix_ to not stick with any name.